android/assets/jsons/translations/Spanish.properties

```diff
@@ -238,14 +238,10 @@ Cultural = Cultural
 Map Shape = Forma del Mapa
 Hexagonal = Hexagonal
 Rectangular = Rectangular
- # Requires translation!
-Height = 
- # Requires translation!
-Width = 
- # Requires translation!
-Radius = 
- # Requires translation!
-Enable Religion = 
+Height = Altura
+Width = Ancho
```

Comment:
anchura? width of the screen/window is what's meant here

Reply:
I think it works like that